en/advanced_config/tuning_the_ecl_ekf.md

```diff
@@ -329,13 +417,13 @@ A good tuning is obtained as follows:
 * Bit position 1 in the [EKF2_AID_MASK](../advanced_config/parameter_reference.md#EKF2_AID_MASK) parameter is true.
 * The quality metric returned by the flow sensor is greater than the minimum requirement set by the [EKF2_OF_QMIN](../advanced_config/parameter_reference.md#EKF2_OF_QMIN) parameter.
 
-<span id="ekf2_extvis"></span>
```

Comment:
FYI, adding spans is something that I'm not trying to avoid unless the heading names is huge. Reason being that vuepress links end up below the heading rather than just above.